#!/usr/bin/env python
# field of view configuration

PACKAGE="pcl_helpers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("lower_limit", double_t, 1, "Lower limit", 0.0, -15.0, 15.0)
gen.add("upper_limit", double_t, 1, "Upper limit", 3.0, -50.0, 50.0)

exit(gen.generate(PACKAGE, "pcl_helpers", "passthrough_filter_node"))
